tests/api/resources/test_solvers.py

```diff
@@ -270,23 +270,31 @@ def tearDownClass(cls):
         with contextlib.suppress(OSError):
             cls.tmpdir.cleanup()
 
-    def test_cache_1_validation(self):
+    def test_default(self):
+        # caching is disabled by default
+        # sapi is queried every time
+        with Solvers.from_config(validate_config_v1(config), history_size=2) as resource:
+            resource.list_solvers(filter='none,+id')
+            self.assertEqual(len(resource.session.history), 1)
+            self.assertEqual(resource.session.history[-1].response.status_code, 200)
+
+            resource.list_solvers(filter='none,+id')
+            self.assertEqual(len(resource.session.history), 2)
+            self.assertEqual(resource.session.history[-1].response.status_code, 200)
```

Comment:
Just to see if I understand this correctly; is 200 is expected here twice since `history_size=2`? Would a third request return a 304 since it'd be using the cache?